xcat-inventory/xcclient/inventory/schema/2.0/1

```diff
@@ -0,0 +1,280 @@
+!!python/dict
```

Comment:
`xcat-inventory/xcclient/inventory/schema/2.0/1` is not valid schema file name.

Reply:
fixed